src/ReduxWPAPI.js

```diff
@@ -99,7 +99,7 @@ export default class ReduxWPAPI {
         ttl = this.adapter.getTTL(request);
       }
 
-      if (ttl !== 0 && !ttl) {
+      if (ttl !== 0 && ttl !== false && !ttl) {
```

Comment:
The cancelation of TTL might be `0` or `false`, any other falsy value is ignored (`null` or `undefined`). But I'll refactor this so its less confusing.